### ISSUE TITLE
Manifold mixup vs. BatchNorm

### ISSUE THREAD

**Issue by chronbird** [yunlu-chen/PointMixup#2]
Hi,
Thanks for the interesting work! I have a question about the manifold mixup. Does it equal to batch normalization when batch size = 2? Is batch norm a non-OA version of manifold mixup? Thanks.

**Comment by yunlu-chen** [comment]
Hi,

Thanks for your question. But manifold mixup is not alike batchnorm at all. In manifold mixup we interpolate the hidden layer representations between random pairs but batchnorm normalizes them in a batch. Hope that answers your question
